crates/transaction-pool/src/pool/best.rs

```diff
@@ -89,6 +113,16 @@ impl<T: TransactionOrdering> Iterator for BestTransactions<T> {
 
     fn next(&mut self) -> Option<Self::Item> {
         loop {
```

Comment:
IIUC this effectively grabs one new transaction from the channel each time next() is called

which may not be correct - i think it may make sense to drain the channel (in another routine w/ an Arc'd lock on this iterator's pool (?)) and add all new transactions in bulk to the pool before yielding

Reply:
wrapping the `try_recv` with a `while` takes care of this concern. resolving